src/main/java/com/atlassian/db/replica/internal/state/ConnectionState.java

```diff
@@ -22,6 +26,7 @@
     private final ConnectionParameters parameters;
     private final Warnings warnings;
     private final StateListener stateListener;
+    private volatile boolean isReplicaInconsistent = false;
```

Comment:
Nits:
* drop the `is` (it's for a getter)
* remove negation: `boolean replicaConsistent`